nsqd/options.go

```diff
@@ -79,6 +79,12 @@ type Options struct {
 	DeflateEnabled  bool `flag:"deflate"`
 	MaxDeflateLevel int  `flag:"max-deflate-level"`
 	SnappyEnabled   bool `flag:"snappy"`
+
+	// contrib
+	// TODO cleanly extend this in the the contrib app
+	DogStatsdAddress  string        `flag:"dogstatsd-address"`
```

Comment:
Not sure how do do this cleanest because brand new to go.  Thinking of NSQDContribOptions that embeds NSQD Options, contrib options will have an option struct for each contrib addon.